src/common/catalog-entities/general-catalog-entities/general-catalog-entities.injectable.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { observable } from "mobx";
+import { generalCatalogEntityInjectionToken } from "./general-catalog-entity-injection-token";
+
+const generalCatalogEntitiesInjectable = getInjectable({
+  id: "general-catalog-entities",
+
+  instantiate: (di) =>
+    observable(di.injectMany(generalCatalogEntityInjectionToken)),
```

Comment:
This does not need to be observable. Only computed.

Reply:
It was observable before. It looks like something that shouldn’t be reactive at all. However we kept it observable simply to make sure that we don’t break anything. Should be changed in different PR to figure out what is really needed and what is not.

Reply:
okay sure, I know for a fact that it doesn't need to be. But it can be a follow up PR if you like.

Reply:
I'll do the change tomorrow since you are absolutely sure. :)

Reply:
Nah just fixed it.

Reply:
@Iku-turso fix didn't do it. Replaced it with: 

**sync-general-catalog-entities.injectable.ts**
```typescript
const generalCatalogEntities = di.injectMany(generalCatalogEntityInjectionToken);

// TODO: This shouldn't be reactive at all but catalogEntityRegistry accepts only reactive sources
const reactiveGeneralCatalogEntities = computed(() => generalCatalogEntities);

return () => {
  catalogEntityRegistry.addComputedSource(
    "lens:general",
    reactiveGeneralCatalogEntities,
  );
};
```

Notice that the requirement for being reactive comes from `catalogEntityRegistry` and in this PR we don't want to dig deeper. Sounds like there should be a way to add source which isn't reactive at all, but as said; lets figure out that in separate PR.

Reply:
@jansav I don't think we should support that use case because a user might accidentally add it and then try to push to that array (or even modify the entities). 

I actually want to get rid of `observableSource` since that complicates things too, but certainly not in this PR.